tiledb/sm/query/result_tile.cc

```diff
@@ -184,9 +186,10 @@ std::string ResultTile::coord_string(uint64_t pos, unsigned dim_idx) const {
     assert(st.ok());
   }
 
-  auto size = next_offset - offset;
```

Comment:
Do we have more details as to why next_offset is smaller than offset? Seems like that would be the source of the problem that we need to debug. While this seems to fix the issue, I'm worried it will introduce more problems...

Reply:
Fair question. The related context is test failures running the R package without having made a change in how it operates or allocates.  The relative change in core triggering this seems to be tied to is the ChunkedBuffer PR #2445: things worked before, and work again after with this change.

Reply:
I am almost sure the smaller next_offset is not from the data set. The data set should be good.  When I was debugging under macOS, it always threw out he "std::bad_allocation" exception. but under windows, sometimes the example(provided by Dirk)  ran successfully, sometimes it threw out the "string too long" exception. Before PR #2445, the chunkedBuffer has a choice for CONTIGUOUS. I am afraid that might be the source of the problem.